aliveim.go

```diff
@@ -56,7 +62,11 @@ func handleAlivePost(rw http.ResponseWriter, request *http.Request) {
 	} else {
 		timer := time.NewTimer(time.Millisecond * time.Duration(aliverequest.Timeout))
 		deviceTimer := DeviceTimer{aliverequest.DeviceID, timer}
+
+		mutex.Lock()
 		timersMap[aliverequest.DeviceID] = deviceTimer
+		mutex.Unlock()
+
 		go deviceTimer.startTimer()
```

Comment:
@alessio what I've seen from the examples is that mutex should be locked right before accessing the variable and right after having accessed it. About the ```startTimer``` I think we are fine because if the Timer is already started it won't be started again, but we should verify this with the official documentation, just to be safe.